utils/__tests__/AsyncIterableTask.test.ts

```diff
@@ -0,0 +1,85 @@
+import { number } from "fp-ts";
+import * as E from "fp-ts/lib/Either";
+import * as TE from "fp-ts/lib/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+import { forEach } from "lodash";
+import * as AI from "../AsyncIterableTask";
```

Comment:
Some `import` aren't required.

Reply:
Done in [eb9bfeb](https://github.com/pagopa/io-functions-services-cache/pull/32/commits/eb9bfeba3948345afca296723fbccd21b409ce56)